### PR TITLE
Fix bug blocking non-networked instances

### LINKED ISSUE #477: Instances without a network fail to start

### PR DESCRIPTION
**Pull request by mandoonandy**
Resolves #477

### CODE REVIEW

File: shakenfist/tasks.py [hunk 1]
```diff
@@ -41,10 +41,12 @@ def json_dump(self):
 # Instance Tasks
 #
 class InstanceTask(QueueTask):
-    def __init__(self, instance_uuid, network=None):
+    def __init__(self, instance_uuid, network=[]):
```
Comment (mikalstill) [inline shakenfist/tasks.py:44]: You can't actually do this in python. You must use None in the method signature and then turn that into an empty list below. https://stackoverflow.com/questions/366422/what-is-the-pythonic-way-to-avoid-default-parameters-that-are-empty-lists has discussion of why, but it comes down to when the interpreter executes the method signature (once on initial parse).

**Review by mikalstill — CHANGES_REQUESTED** [review]

File: shakenfist/tasks.py (hunk 1 above)
Comment (mandoonandy) [reply shakenfist/tasks.py:44]: Very good point.

File: deploy/shakenfist_ci/tests/test_cirros.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
+from shakenfist_ci import base
+
+
+class TestCirros(base.BaseNamespacedTestCase):
+    '''Make sure instances boot under various configurations.
```
Comment (mikalstill) [inline deploy/shakenfist_ci/tests/test_cirros.py:5]: Should be """, but that's because I am a people person.

**Review by mikalstill — APPROVED** [review]
LGTM, except for the people person bit.

**Pull request merged**